pom.xml

```diff
@@ -1,470 +1,473 @@
-				<configuration>
-					<header>com/mycila/maven/plugin/license/templates/LGPL-3.txt</header>
-					<properties>
-						<owner>Agile Knowledge Engineering and Semantic Web (AKSW)
-							(usbeck@informatik.uni-leipzig.de)</owner>
-					</properties>
-					<excludes>
-						<exclude>**/README</exclude>
-						<exclude>*.log</exclude>
-						<exclude>*.html</exclude>
-						<exclude>start.sh</exclude>
-						<exclude>**/LICENSE</exclude>
-						<exclude>documentation/**</exclude>
-						<exclude>repository/**</exclude>
-						<exclude>gerbil_data/**</exclude>
-						<exclude>src/main/resources/**</exclude>
-						<exclude>src/main/properties/**</exclude>
-						<exclude>src/main/webapp/**</exclude>
-						<exclude>src/test/resources/**</exclude>
-					</excludes>
-				</configuration>
-			</plugin>
-			<!-- Maven Exec Plugin -->
-			<plugin>
-				<groupId>org.codehaus.mojo</groupId>
-				<artifactId>exec-maven-plugin</artifactId>
-				<version>1.5.0</version>
-			</plugin>
-			<!-- Javadoc plugin for generating documentation -->
-			<plugin>
-				<groupId>org.apache.maven.plugins</groupId>
-				<artifactId>maven-javadoc-plugin</artifactId>
-				<version>2.10.1</version>
-				<configuration>
-					<configuration>
-						<show>private</show>
-						<nohelp>true</nohelp>
-						<quiet>true</quiet>
-						<!-- added for generating javadoc with java 1.8 -->
-						<additionalparam>-Xdoclint:none</additionalparam>
-					</configuration>
-				</configuration>
-				<executions>
-					<execution>
-						<phase>package</phase>
-						<goals>
-							<!-- <goal>jar</goal> -->
-						</goals>
-					</execution>
-				</executions>
-			</plugin>
-			<!-- Source plugin for creating source file jar -->
-			<plugin>
-				<groupId>org.apache.maven.plugins</groupId>
-				<artifactId>maven-source-plugin</artifactId>
-				<version>2.4</version>
-				<executions>
-					<execution>
-						<id>attach-sources</id>
-						<goals>
-							<goal>jar</goal>
-						</goals>
-					</execution>
-				</executions>
-			</plugin>
-		</plugins>
-
-		<resources>
-			<resource>
-				<directory>src/main/resources</directory>
-			</resource>
-			<resource>
-				<!-- Note that the properties files will be filtered. If mvn finds known 
-					place holders they will be replaced -->
-				<directory>src/main/properties</directory>
-				<filtering>true</filtering>
-			</resource>
-		</resources>
-	</build>
-	<distributionManagement>
-		<repository>
-			<id>maven.aksw.internal</id>
-			<url>http://maven.aksw.org/archiva/repository/internal</url>
-		</repository>
-		<snapshotRepository>
-			<id>maven.aksw.snapshots</id>
-			<name>AKSW Snapshot Repository</name>
-			<url>http://maven.aksw.org/archiva/repository/snapshots</url>
-		</snapshotRepository>
-	</distributionManagement>
-	<packaging>war</packaging>
-</project>
+<!-- This file is part of General Entity Annotator Benchmark. General Entity 
+	Annotator Benchmark is free software: you can redistribute it and/or modify 
```

Comment:
@Suganya31 
Looks like formatting of whole file has been changed. This might be problematic for someone who's trying to compare changes in the commit history. I believe this problem can be solved by looking into the whitespace settings of the local git.